desktop/apps/artwork/components/commercial/templates/index.jade

```diff
@@ -1,5 +1,5 @@
 .artwork-commercial( class='js-artwork-commercial' )
-  form.artwork-commercial__form
+  form.artwork-commercial__form( style= !artwork.is_for_sale ? "padding-top: 0" : "" )
```

Comment:
There could be some bottom padding, but having a zero padding would break the style when the question section is shown. If there's nothing below the button, then we shouldn't have to care about button padding at all. So I think just specifying a top padding is a little safer.